iceoryx_hoofs/test/moduletests/test_semaphore_module.cpp

```diff
@@ -138,7 +138,7 @@ TEST_F(SemaphoreCreate_test, OpenNamedSemaphore)
 {
     ::testing::Test::RecordProperty("TEST_ID", "349cdf0d-987e-4e2f-aa35-98a40fdf979b");
     auto semaphore = iox::posix::Semaphore::create(iox::posix::CreateNamedSemaphore, "/fuuSem", S_IRUSR | S_IWUSR, 10);
-    auto semaphore2 = iox::posix::Semaphore::create(iox::posix::OpenNamedSemaphore, "/fuuSem", S_IRUSR | S_IWUSR);
+    auto semaphore2 = iox::posix::Semaphore::create(iox::posix::OpenNamedSemaphore, "/fuuSem", O_CREAT);
```

Comment:
Hint: Another bug uncovered by a unix posix system. The second parameter are the `oflags` and not `mode_t`.
By accident the `mode_t` values were compatible to the `oflag` values  in this test. But who knows what they meant here.